python_modules/dagster/dagster/components/core/decl.py

```diff
@@ -133,8 +133,13 @@ def _process_attributes_with_enriched_validation_err(
         if attributes_position_tree
         else nullcontext()
     ):
+        if issubclass(model_cls, EmptyAttributesModel):
+            raise DagsterInvalidDefinitionError(
+                "Component defines no model class, but attributes were provided. This error can be resolved by "
```

Comment:
> defines no model class

i find this a bit confusing since it inherits `Model` in this example, should it be more like "`Component` is not resolvable from yaml" or something? 

>  error can be resolved 

nit: worth substituting "resolved" for describing handling the problem to avoid polluting our specific use of it for this system